docs/development/devtools.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Devtools
+description: ✨ What devtools are and how to use them.
```

Comment:
This is documentation for developers, so if they don't know what devtools are, it is not our job to teach them 😅

```suggestion
description: ✨ How to use Spicetify devtools.
```